lib/realtime/tenants/connect.ex

```diff
@@ -66,9 +84,9 @@ defmodule Realtime.Tenants.Connect do
       case res do
         {:ok, conn} ->
           :syn.update_registry(__MODULE__, tenant_id, fn _pid, meta -> %{meta | conn: conn} end)
-          state = %{state | db_conn_reference: Process.monitor(conn)}
+          state = %{state | db_conn_reference: Process.monitor(conn), db_conn_pid: conn}
 
-          {:ok, state}
+          {:ok, state, {:continue, :setup_connected_users}}
```

Comment:
I would either send a `:check_connected_users` message or wrap `Process.send_after` in a function and call it. Currently, a similar code exists three times.